src/main/java/de/retest/ui/descriptors/Attributes.java

```diff
@@ -109,7 +109,7 @@ public String toString() {
 
 	// let's not spread the eclipse moxy bug out into the rest of the system but
 	// instead convert the map here:
-	public Map<String, ? extends Serializable> getMap() {
+	public Map<String, Serializable> getMap() {
```

Comment:
I also don't know what this comment means in this context (let's discuss tomorrow)?